### DISCUSSION TITLE
Returning redirect in getServerSideProps return Additional Key error

### DISCUSSION THREAD

**Discussion by jackchengck** [vercel/next.js#20340] [Help]
Next Version: 10.0.3

I created a page solely to test out this problem.
It is using the example code on the doc.

```
export default function testRedirect() {
  return (
    <div>
      <h1>This is Redirect Page</h1>
    </div>
  );
}

export async function getServerSideProps(context) {
  return {
    redirect: {
      destination: "/",
      permanent: false,
    },
  };
}
```

However, I am still getting this page
<img width="1400" alt="螢幕截圖 2020-12-20 上午7 55 39" src="https://user-images.githubusercontent.com/13392949/102702024-c1ded300-4298-11eb-8a59-af99baedb753.png">

**Comment by ijjk** [comment]
Hi, can you provide a repo with reproduction where this is occurring? We have tests covering this feature which are passing, and I'm not able to reproduce this manually on `v10.0.3` of Next.js.
